aspnetcore/blazor/forms/input-components.md

```diff
@@ -67,6 +67,18 @@ Input components provide default behavior for validating when a field is changed
 
 Some components include useful parsing logic. For example, <xref:Microsoft.AspNetCore.Components.Forms.InputDate%601> and <xref:Microsoft.AspNetCore.Components.Forms.InputNumber%601> handle unparseable values gracefully by registering unparseable values as validation errors. Types that can accept null values also support nullability of the target field (for example, `int?` for a nullable integer).
 
+:::moniker range=">= aspnetcore-9.0"
+
+The <xref:Microsoft.AspNetCore.Components.Forms.InputNumber%601> component supports overriding the `type` attribute. For example, specify an [`<input>` element of `type="range"`](https://developer.mozilla.org/docs/Web/HTML/Element/input/range) to create a range input that supports model binding and form validation, typically rendered as a slider or dial control rather than a text box:
+
```

Comment:
I wonder if, rather than noting that `<InputNumber>` has an overridable `"type"`, we just mention that it supports specifying `type="range"` specifically. I can't think of another input type that customers would want to specify other than `"range"`, and I don't necessarily want to claim that we support anything other than `"range"` (because that's not tested and might have some weird behavior).

> For example, specify an [`<input>` element of `type="range"`](https://developer.mozilla.org/docs/Web/HTML/Element/input/range) to create a range input that supports model binding and form validation

Where does the customer specify an `<input>` element? They're using `<InputNumber>` right? Should this instead say:

> Specify the [`type="range"` attribute](https://developer.mozilla.org/docs/Web/HTML/Element/input/range) to create a range input that supports model binding and form validation

?

Reply:
I'm sorry about this. The wording is blown because I'm trying to get off of this PC 😆 ... a bit of 🦖🧠🔥hit me today.

Hopefully, I'll get it right on the next commit. 🤞🍀